packages/accordions/src/elements/stepper/components/Label.tsx

```diff
@@ -12,10 +12,15 @@ import { StyledLabel, StyledLabelText, StyledIcon, StyledIconFlexContainer } fro
 import { useStepContext, useStepperContext } from '../../../utils';
 
 interface ILabel {
+  /** Used as the visual object for the stepper */
```

Comment:
If we strive to follow grammar conventions, perhaps something like:

> Replaces the Stepper number with an icon

You can observe some key prop affects by referencing website documentation or the "Controls" tab in storybook:
- https://garden.zendesk.com/components/stepper#icons
- https://zendeskgarden.github.io/react-components/storybook/?path=/story/components-accordions-stepper--default

@vsrixyz I think we want to refer to all component names with capitalization, right?

Reply:
Fixed.